### ISSUE TITLE
SyntaxError in Python 2.7

### ISSUE THREAD

**Issue by HayaoSuzuki** [rubik/radon#152]
I have a problem when I try to use Radon 2.3.0 with Python 2.7.15 as below:

```
(venv) radon cc hoge.py
Traceback (most recent call last):
  File "/venv/bin/radon", line 11, in <module>
    sys.exit(main())
  File "/venv/lib/python2.7/site-packages/radon/__init__.py", line 10, in main
    from radon.cli import program, log_error
  File "/venv/lib/python2.7/site-packages/radon/cli/__init__.py", line 9, in <module>
    from radon.cli.harvest import CCHarvester, RawHarvester, MIHarvester, HCHarvester
  File "/venv/lib/python2.7/site-packages/radon/cli/harvest.py", line 307
    yield from hal_report_to_terminal(report, 1)
             ^
SyntaxError: invalid syntax
```

My envrionments are here:

```
(venv) python -V
Python 2.7.15
(venv) pip freeze
colorama==0.3.9
configparser==3.5.0
enum34==1.1.6
flake8==3.5.0
flake8-polyfill==1.0.2
mando==0.6.4
mccabe==0.6.1
pycodestyle==2.3.1
pyflakes==1.6.0
radon==2.3.0
six==1.11.0
```

**Comment by rubik** [comment]
Thanks, this is fixed in version 2.3.1 which has been uploaded to PyPI.

**Comment by HayaoSuzuki** [comment]
Thank you so much for fixing. It's working.
